benchmarks/StiffODE/VanDerPol.jmd

```diff
@@ -139,7 +139,7 @@ setups = [Dict(:alg=>Rosenbrock23()),
           Dict(:alg=>Rosenbrock23(), :prob_choice => 2),
           Dict(:alg=>Rodas3()),
           Dict(:alg=>TRBDF2()),
-          Dict(:alg=>rodas()),
```

Comment:
why? We want to benchmark against Hairer.

Reply:
it was hanging

Reply:
Add a note about that. There's always a notes on why some methods weren't benchmarked.